pkg/handlers/webhook/webhook.go

```diff
@@ -50,7 +50,17 @@ type Webhook struct {
 
 // WebhookMessage for messages
 type WebhookMessage struct {
-	Text string `json:"text"`
+	EventMeta EventMeta `json:"eventmeta"`
+	Message   string    `json:"message"`
```

Comment:
This should remain as 'Text` so it doesn't break existing webhooks.
```suggestion
	Text string `json:"text"`
```

Reply:
resolved